File: core/src/main/java/com/a65apps/core/interactors/reminders/BirthdayReminderInteractor.java
package com.a65apps.core.interactors.reminders;

import java.util.GregorianCalendar;

import io.reactivex.rxjava3.annotations.NonNull;

public interface BirthdayReminderInteractor {
    boolean setBirthdayReminder(@NonNull final String personId, @NonNull String fullName, @NonNull String date);
    boolean unsetBirthdayReminder(@NonNull final String personId);
    boolean isReminderOn(@NonNull final String personId);
}

File: core/src/main/java/com/a65apps/core/interactors/reminders/BirthdayReminderInteractor.kt
package com.a65apps.core.interactors.reminders

interface BirthdayReminderInteractor {
    fun setBirthdayReminder(personId: String, fullName: String, date: String): Boolean
    fun unsetBirthdayReminder(personId: String): Boolean
    fun isReminderOn(personId: String): Boolean
}
